### PR TITLE
Rubocop addition

### PR DESCRIPTION
**Pull request by fahlmant**

### CODE REVIEW

**Comment by armiller** [comment]
:+1:

**Comment by armiller** [comment]
@fahlmant can you rebase?

**Comment by fahlmant** [comment]
@armiller Done

File: recipes/default.rb [hunk 1]
```diff
@@ -49,16 +49,16 @@
       end
     end
   end
+end
```
Comment (jordane) [inline recipes/default.rb]: I think this end is in the wrong spot after the rebase. @armiller?

Comment (armiller) [inline recipes/default.rb]: yes that looks wrong. The unless block should encapsulate everything.

Comment (fahlmant) [inline recipes/default.rb]: @armiller Should line 44 shift to spaces to the left?

**Comment by fahlmant** [comment]
@armiller ?

File: recipes/default.rb [hunk 2]
```diff
@@ -48,17 +48,16 @@
...
+  supports restart: true
+  action :nothing
+end
+
+node['ucarp']['bonded_interfaces'].each do |interface|
+  execute "flush ip on #{interface}" do
+    command "ip addr flush dev #{interface}"
+    action :nothing
   end
 end
```
Comment (armiller) [inline recipes/default.rb:64]: The service and node['ucarp'] blocks need to be under the unless block. So these need to be indented with an additional `end` on the bottom.

**Comment by fahlmant** [comment]
@armiller Look good now?

**Comment by armiller** [comment]
@fahlmant that looks correct. :+1:

**Pull request merged**